### PR TITLE
[Docs] refactor get_started.md

### PR DESCRIPTION
**Pull request by Jiahao000**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/get_started.md [hunk 1]
```diff
@@ -3,24 +3,28 @@
...
-  - [Using multiple MMSelfSup versions](#using-multiple-mmselfsup-versions)
+  - [Using Multiple MMSelfSup Versions](#using-multiple-mmselfsup-versions)
 
 ## Prerequisites
 
-In this section we demonstrate how to prepare an environment with PyTorch.
+In this section, we demonstrate how to prepare an environment with PyTorch.
 
-MMselfSup works on Linux (Windows and macOS are not officially supported). It requires Python 3.6+, CUDA 9.2+ and PyTorch 1.5+.
+MMSelfSup works on Linux (Windows and macOS are not officially supported). It requires Python 3.6+, CUDA 9.2+ and PyTorch 1.5+.
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
MMSelfSup works on Linux (Windows and macOS are not officially supported). It requires Python 3.6+, PyTorch 1.6+.
```

File: docs/en/get_started.md [hunk 2]
```diff
@@ -47,39 +51,55 @@ conda install pytorch torchvision cpuonly -c pytorch
...
-### Best Practices
+### Best practices
 
-**Step 0.** Install [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
+**Step 0.** Install [MMEngine](https://github.com/open-mmlab/mmengine) and [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
 
 ```shell
 pip install -U openmim
-mim install mmcv-full
+mim install mmengine "mmcv-full>=2.0rc0"
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
mim install mmengine "mmcv>=2.0.0rc1"
```

File: docs/en/get_started.md [hunk 3]
```diff
@@ -47,39 +51,55 @@ conda install pytorch torchvision cpuonly -c pytorch
...
-pip install mmselfsup
+git checkout dev-1.x
+```
+
+#### Install as a Python package
+
+Just install with pip.
+
+```shell
+pip install "mmselfsup>=1.0rc0"
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
pip install "mmselfsup>=1.0.0rc0"
```

File: docs/en/get_started.md [hunk 4]
```diff
@@ -141,54 +161,53 @@ Please make sure the GPU driver satisfies the minimum version requirements. See
...
 
-**Important:** Make sure you've installed the [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker).
+To install MMCV with pip instead of MIM, please follow [MMCV installation guides](https://mmcv.readthedocs.io/en/latest/get_started/installation.html). This requires manually specifying a find-url based on PyTorch version and its CUDA version.
 
-Run the following cmd:
+For example, the following command install mmcv-full built for PyTorch 1.10.x and CUDA 11.3.
 
 ```shell
-docker run --gpus all --shm-size=8g -it -v {DATA_DIR}:/workspace/mmselfsup/data mmselfsup:torch1.10.0-cuda11.3-cudnn8 /bin/bash
+pip install "mmcv-full>=2.0rc0" -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.10/index.html
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
pip install "mmcv-full>=2.0.0rc1" -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.10/index.html
```

File: docs/en/get_started.md [hunk 5]
```diff
@@ -141,54 +161,53 @@ Please make sure the GPU driver satisfies the minimum version requirements. See
...
 [Google Colab](https://research.google.com/) usually has PyTorch installed,
 thus we only need to install MMCV and MMSeflSup with the following commands.
 
-**Step 0.** Install [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
+**Step 0.** Install [MMEngine](https://github.com/open-mmlab/mmengine) and [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
 
 ```shell
 !pip3 install openmim
-!mim install mmcv-full
+!mim install mmengine "mmcv-full>=2.0rc0"
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
!mim install mmengine "mmcv>=2.0.0rc0"
```

File: docs/en/get_started.md [hunk 6]
```diff
@@ -47,39 +51,55 @@ conda install pytorch torchvision cpuonly -c pytorch
...
+pip install "mmselfsup>=1.0rc0"
 ```
 
 ### Verify the installation
 
-To verify whether MMSelfSup is installed correctly, we can run the following sample code to initialize a model and inference a demo image.
+To verify whether MMSelfSup is installed correctly, you can run the following sample code to initialize a model and inference a demo image.
 
 ```python
 import torch
```
Comment (fangyixiao18) [inline docs/en/get_started.md:114]: i think we could use version check instead of it
E.g.
```python
import mmselfsup
print(mmselfsup.__version__)
```

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

File: docs/en/get_started.md [hunk 7]
```diff
@@ -3,24 +3,28 @@
...
 ## Prerequisites
 
-In this section we demonstrate how to prepare an environment with PyTorch.
+In this section, we demonstrate how to prepare an environment with PyTorch.
 
-MMselfSup works on Linux (Windows and macOS are not officially supported). It requires Python 3.6+, CUDA 9.2+ and PyTorch 1.5+.
+MMSelfSup works on Linux (Windows and macOS are not officially supported). It requires Python 3.6+, CUDA 9.2+ and PyTorch 1.6+.
 
+```{note}
 If you are experienced with PyTorch and have already installed it, just skip this part and jump to the [next section](#installation). Otherwise, you can follow these steps for the preparation.
```
Comment (soonera) [inline docs/en/get_started.md]: It's not a link.
```{note}
[next section](#installation)
```

File: docs/en/get_started.md [hunk 8]
```diff
@@ -141,54 +170,54 @@ Please make sure the GPU driver satisfies the minimum version requirements. See
...
 
-**Important:** Make sure you've installed the [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker).
+To install MMCV with pip instead of MIM, please follow [MMCV installation guides](https://mmcv.readthedocs.io/en/latest/get_started/installation.html). This requires manually specifying a find-url based on PyTorch version and its CUDA version.
 
-Run the following cmd:
+For example, the following command install mmcv-full built for PyTorch 1.10.x and CUDA 11.3.
 
 ```shell
-docker run --gpus all --shm-size=8g -it -v {DATA_DIR}:/workspace/mmselfsup/data mmselfsup:torch1.10.0-cuda11.3-cudnn8 /bin/bash
+pip install mmcv-full>=2.0.0rc1 -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.10/index.html
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
pip install mmcv-full>=2.0.0rc1 -f https://download.openmmlab.com/mmcv/dist/cu116/torch1.12.0/index.html
```

File: docs/en/get_started.md [hunk 9]
```diff
@@ -112,21 +141,21 @@ loss = model.forward_train(image, label)
...
-The [Best Practices](#best-practices) is for basic usage, if you need to evaluate your pre-training model with some downstream tasks such as detection or segmentation, please also install [MMDetection](https://github.com/open-mmlab/mmdetection) and [MMSegmentation](https://github.com/open-mmlab/mmsegmentation).
+The [Best practices](#best-practices) is for basic usage. If you need to evaluate your pre-trained model with some downstream tasks such as detection or segmentation, please also install [MMDetection](https://github.com/open-mmlab/mmdetection) and [MMSegmentation](https://github.com/open-mmlab/mmsegmentation).
 
-If you don't run MMDetection and MMSegmentation benchmark, it is unnecessary to install them.
+If you don't run MMDetection and MMSegmentation benchmarks, it is unnecessary to install them.
 
 You can simply install MMDetection and MMSegmentation with the following command:
 
 ```shell
 pip install mmdet mmsegmentation
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
pip install 'mmdet>=3.0.0rc0' 'mmsegmentation>=1.0.0rc0'
```

File: docs/en/get_started.md [hunk 10]
```diff
@@ -141,54 +170,54 @@ Please make sure the GPU driver satisfies the minimum version requirements. See
...
-# build an image with PyTorch 1.6.0, CUDA 10.1, CUDNN 7.
-docker build -f ./docker/Dockerfile --rm -t mmselfsup:torch1.10.0-cuda11.3-cudnn8 .
-```
+MMCV contains C++ and CUDA extensions, thus depending on PyTorch in a complex way. MIM solves such dependencies automatically and makes the installation easier. However, it is not a must.
 
-**Important:** Make sure you've installed the [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker).
+To install MMCV with pip instead of MIM, please follow [MMCV installation guides](https://mmcv.readthedocs.io/en/latest/get_started/installation.html). This requires manually specifying a find-url based on PyTorch version and its CUDA version.
 
-Run the following cmd:
+For example, the following command install mmcv-full built for PyTorch 1.10.x and CUDA 11.3.
```
Comment (fangyixiao18) [inline docs/en/get_started.md]: update version as mentioned in feishu group

File: docs/en/get_started.md [hunk 11]
```diff
@@ -141,54 +170,54 @@ Please make sure the GPU driver satisfies the minimum version requirements. See
...
+pip install mmcv-full>=2.0.0rc1 -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.10/index.html
 ```
 
-`{DATA_DIR}` is your local folder containing all these datasets.
+### Install on CPU-only platforms
+
+MMSelfSup can be built for CPU only environment. In CPU mode, you can train, test or inference a model.
+
+Some functionalities are gone in this mode, usually GPU-compiled ops. But don't
+worry, almost all models in MMClassification don't depends on these ops.
```
Comment (soonera) [inline docs/en/get_started.md]:
```suggestion
worry, almost all models in MMSelfsup don't depends on these ops.
```

File: docs/en/get_started.md [hunk 12]
```diff
@@ -141,54 +170,54 @@ Please make sure the GPU driver satisfies the minimum version requirements. See
 Installing CUDA runtime libraries is enough if you follow our best practices, because no CUDA code will be compiled locally. However if you hope to compile MMCV from source or develop other CUDA operators, you need to install the complete CUDA toolkit from NVIDIA's [website](https://developer.nvidia.com/cuda-downloads), and its version should match the CUDA version of PyTorch. i.e., the specified version of cudatoolkit in `conda install` command.
 ```
 
-#### Install MMCV without MIM
-
-MMCV contains C++ and CUDA extensions, thus depending on PyTorch in a complex way. MIM solves such dependencies automatically and makes the installation easier. However, it is not a must.
+#### Install MMEngine without MIM
 
-To install MMCV with pip instead of MIM, please follow [MMCV installation guides](https://mmcv.readthedocs.io/en/latest/get_started/installation.html). This requires manually specifying a find-url based on PyTorch version and its CUDA version.
+To install MMEngine with pip instead of MIM, please follow [MMEngine installation guides](https://mmcv.readthedocs.io/en/latest/get_started/installation.html).
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
To install MMEngine with pip instead of MIM, please follow [MMEngine installation guides](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/get_started/installation.md).
```

File: docs/en/get_started.md [hunk 13]
```diff
@@ -141,54 +170,54 @@ Please make sure the GPU driver satisfies the minimum version requirements. See
...
+#### Install MMCV without MIM
 
-```shell
-# build an image with PyTorch 1.6.0, CUDA 10.1, CUDNN 7.
-docker build -f ./docker/Dockerfile --rm -t mmselfsup:torch1.10.0-cuda11.3-cudnn8 .
-```
+MMCV contains C++ and CUDA extensions, thus depending on PyTorch in a complex way. MIM solves such dependencies automatically and makes the installation easier. However, it is not a must.
 
-**Important:** Make sure you've installed the [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker).
+To install MMCV with pip instead of MIM, please follow [MMCV installation guides](https://mmcv.readthedocs.io/en/latest/get_started/installation.html). This requires manually specifying a find-url based on PyTorch version and its CUDA version.
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
To install MMCV with pip instead of MIM, please follow [MMCV installation guides](https://mmcv.readthedocs.io/en/2.x/get_started/installation.html). This requires manually specifying a find-url based on PyTorch version and its CUDA version.
```

File: docs/en/get_started.md [hunk 14]
```diff
@@ -197,19 +226,38 @@ thus we only need to install MMCV and MMSeflSup with the following commands.
...
+```shell
+docker run --gpus all --shm-size=8g -it -v {DATA_DIR}:/workspace/mmselfsup/data mmselfsup:torch1.10.0-cuda11.3-cudnn8 /bin/bash
+```
+
+`{DATA_DIR}` is your local folder containing all these datasets.
+
 ### Trouble shooting
 
-If you have some issues during the installation, please first view the [FAQ](faq.md) page.
+If you have some issues during the installation, please first view the [FAQ](https://github.com/open-mmlab/mmselfsup/blob/dev-1.x/docs/en/notes/faq.md) page.
```
Comment (fangyixiao18) [inline docs/en/get_started.md]:
```suggestion
If you have some issues during the installation, please first view the [FAQ](notes/faq.md) page.
```

**Review by fangyixiao18 — APPROVED** [review]
